### ISSUE TITLE
Oro's stomach stretches when drinking.

### ISSUE THREAD

**Issue by MarianaBoni** [deathlyrage/theisle-bugs#288]
Patch: 4691
Server: Hosted

Oro's stomach stretches significantly when it drinks.
https://youtu.be/79rNlFKEN1U

**Comment by dinosauriac** [comment]
Yeah we're pretty sure this is related to the vertex stretching problems that introduced.. 
https://github.com/deathlyrage/theisle-bugs/issues/260

Last I checked the model's going to get tweaked again to fix the issue (when there's a free moment).
